2.x-activity-module/mod/aspirelists/yui/build/moodle-mod_aspirelists-inline_display/moodle-mod_aspirelists-inline_display-debug.js

```diff
@@ -27,6 +28,11 @@ NS.toggle_inline_list = function(e)
         {
             if(n.getStyle('display') === 'none')
             {
+                if(n.getAttribute('src') === ''){
```

Comment:
I'm guessing you've already tested this, but is `src` really set to the empty string, rather than just a falsy value?